dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -1241,8 +1241,8 @@ def _update_dependency_line_with_new_version(
     new_constraint = f'"{provider_package_name}>={new_version}"'
     updated_line = line.replace(old_constraint, new_constraint)
 
-    # Remove the "# use next version" comment after upgrading
-    updated_line = updated_line.replace(" # use next version", "")
+    # remove the comment starting with '# use next version' (and anything after it) and rstrip spaces
+    updated_line = re.sub(r"#\s*use next version.*$", "", updated_line).rstrip()
```

Comment:
I had two cases where comma was added after the comment - it seems it is eeasy to do it when adding `# use next version`